pkg/filter/filter.go

```diff
@@ -145,6 +146,16 @@ func Owned(obj client.Object) bool {
 	return false
 }
 
+// Check if the daemonset is owned by SRO and is kernel affine
+func IsOwnedKernelAffineObject(obj client.Object) bool {
+
+	if Owned(obj) && kernel.IsObjectAffine(obj) {
```

Comment:
Thank you for considering this @Veritasosrb. This could now be simplified to
```suggestion
	return Owned(obj) && kernel.IsObjectAffine(obj)
```
But I would in fact remove this function entirely and move that `if` to the calling code. WDYT?

Reply:
Yes that would be better. Thanks for pointing out. Done.